docroot/sites/all/themes/ilr_theme/scss/layout/_ilr-at-70-homepage-layout.scss

```diff
@@ -52,7 +51,7 @@ body.subsite-front {
           @include grid-span(7,6);
         }
         ul {
-          left: 42%;
```

Comment:
Here is the link to the gist:
https://gist.github.com/fjp2cornell/980b7ada91d2b3b4d64b
I added a comment about the current location of the ul.